### DISCUSSION TITLE
Google Voice

### DISCUSSION THREAD

**Discussion by OmegaMetor** [fonoster/yaps#228] [Q&A] [answered]
Is it possible to use google voice as the voip provider for fonoster? If so, how?

**Comment by psanders** [comment, marked as answer]
Hi @OmegaMetor 

I've never tried this myself, but it seems that it violates Google's Voice Acceptable Use Policy. 

If you are looking for an inexpensive way to make/receive calls I could recommend a few VoIP providers. You could also look into something like [GoIP](https://en.wikipedia.org/wiki/GoIP).
